mep/people/management/commands/export_creators.py

```diff
@@ -12,12 +12,29 @@
 
 
 class Command(ExportMemberCommand):
-    """Export member data."""
+    """Export creator data."""
+
+    csv_fields = [
+        "id",  # slug will be used (unlike members, no URI to use)
+        "name",
+        "sort_name",
+        "title",
+        "gender",
+        "is_organization",
+        "birth_year",
+        "death_year",
+        "viaf_url",
+        "wikipedia_url",
+        # related country
+        "nationalities",
+        # generic
+        "notes",
+        "updated",
+    ]
 
     def get_queryset(self):
         """filter to creators"""
-        creator_ids = {c.person_id for c in Creator.objects.all()}
-        return Person.objects.filter(pk__in=creator_ids)
+        return Person.objects.filter(creator__isnull=False)
```

Comment:
pretty sure you need the `.distinct()`; did you test with this and see what you get?

Reply:
woops, I forgot to include that, thanks. It's def needed: with `distinct` we have 2532 creators from queryset to export and without it we have 6479.